internal/provisioner/plain/plain.go

```diff
@@ -19,9 +19,9 @@ import (
 
 const (
 	// ProvisionerID is the unique plain provisioner ID
-	ProvisionerID = "core-rukpak-io-plain"
-
-	manifestsDir = "manifests"
+	ProvisionerID            = "core-rukpak-io-plain"
+	customResourceDefinition = "CustomResourceDefinition"
+	manifestsDir             = "manifests"
```

Comment:
_total_ nit: group plain provisioner specific constants separately from sort of general helper constants?
```suggestion
	ProvisionerID = "core-rukpak-io-plain"
	manifestsDir  = "manifests"
	
	customResourceDefinition = "CustomResourceDefinition"
```

Also, IMO since there's just one place we use this constant, maybe it's okay to just hardcode `"CustomResourceDefinition"` in the comparison and drop the constant?

Reply:
Yup, makes sense to me. Will update. I was hoping to find an external const I could use in the apiextensionsv1 pkg but came up empty :(